### ISSUE TITLE
[BUG/ISSUE] Error in fast_jx_mod.F90 terminates nested grid simulation

### ISSUE THREAD

**Issue by tianjialiu** [geoschem/geos-chem#444]
My nested grid simulation suddenly stopped with an error in fast_jx_mod.F90 on December 13, 2016. HEMCO.log and GC.log did not indicate any errors, but the slurm error output did indicate an error:

```
At line 1587 of file /n/home00/tianjialiu/GC/rundirs/merra2_05x0625_tropchem_as_nofire/CodeDir/GeosCore/fast_jx_mod.F90 (unit = 6, file = 'stdout')
Fortran runtime error: Expected REAL for item 4 in formatted transfer, got INTEGER
(A,2I5,F9.2)
       ^

Error termination. Backtrace:
#0  0x2b28cb16c029 in require_type
	at ../.././libgfortran/io/transfer.c:1266
#1  0x2b28cb16eaba in require_type
	at ../.././libgfortran/io/transfer.c:1258
#2  0x2b28cb16eaba in formatted_transfer_scalar_write
	at ../.././libgfortran/io/transfer.c:2032
#3  0x2b28cb16f06c in formatted_transfer
	at ../.././libgfortran/io/transfer.c:2272
#4  0x467825 in extral
	at /n/home00/tianjialiu/GC/rundirs/merra2_05x0625_tropchem_as_nofire/CodeDir/GeosCore/fast_jx_mod.F90:1587
#5  0x467825 in __fast_jx_mod_MOD_photo_jx
	at /n/home00/tianjialiu/GC/rundirs/merra2_05x0625_tropchem_as_nofire/CodeDir/GeosCore/fast_jx_mod.F90:3640
#6  0x46f22b in __fast_jx_mod_MOD_fast_jx._omp_fn.0
	at /n/home00/tianjialiu/GC/rundirs/merra2_05x0625_tropchem_as_nofire/CodeDir/GeosCore/fast_jx_mod.F90:406
#7  0x2b28cb6cd8fd in gomp_thread_start
	at ../.././libgomp/team.c:120
#8  0x2b28cbd42dd4 in ???
#9  0x2b28cc054eac in ???
#10  0xffffffffffffffff in ???
36637.29user 207.16system 55:56.91elapsed 1097%CPU (0avgtext+0avgdata 28275292maxresident)k
5414320inputs+304144outputs (34major+31406538minor)pagefaults 0swaps
srun: error: holy7c20204: task 0: Exited with exit code 2
```

**Comment by yantosca** [comment]
Thanks for writing. This error:
```
At line 1587 of file /n/home00/tianjialiu/GC/rundirs/merra2_05x0625_tropchem_as_nofire/CodeDir/GeosCore/fast_jx_mod.F90 (unit = 6, file = 'stdout')
Fortran runtime error: Expected REAL for item 4 in formatted transfer, got INTEGER
(A,2I5,F9.2)
```
means that the model expected to read a character value, 2 integer values of 5 spaces, and then a floating-point value of 9 spaces total and 2 decimal spaces.   But as you can see I think it is trying to print out 3 integer values:
```f90
#else
    JTOTL    = L2X + 2
    do L2 = L2X,1,-1
       JTOTL  = JTOTL + JXTRA(L2)
       if (JTOTL .gt. NX/2)  then
          write(6,'(A,2I5,F9.2)') 'N_/L2_/L2-cutoff JXTRA:',NX,L2X,L2
          do L = L2,1,-1
             JXTRA(L) = 0
          enddo
          go to 10
       endif
    enddo
10  continue
#endif
```
My guess is that this is IF block of the code normally doesn't get executed, so that error very rarely happens.  The solution would be to change the FORMAT statement to
```F90
WRITE(6, '(A, 3I5)') ... etc...
```
We can add this to 13.0.0.

**Comment by yantosca** [comment]
This issue is now resolved with commit 868248b01c47a5d7dc7cf4206d36c9875ddab76c.

**Comment by yantosca** [comment]
The commit mentioned in the prior comment was never merged into the mainstream.  The correct commit is now: 0acea73c1ad965d27db3447e0e7c13dc1bde776f
